src/shared/api/base.ts

```diff
@@ -122,7 +121,7 @@ async function executeFetch<T = unknown>(
     };
   } catch (e) {
     if (e instanceof Error && e.name === 'TypeError') {
-      await delay(RETRY_BACKOFF[retry]);
```

Comment:
This change doesn't seem consistent with the old backoff strategy:

```ts
export const RETRY_BACKOFF: { [key: number]: number } = {
  5: 10_000,
  4: 20_000,
  3: 30_000,
  2: 30_000,
  1: 30_000,
};
```

Wouldn't this just increase by 10000 ms each time after the update?